modules/openshift-developer-cli-using-command-completion.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/configuring-the-odo-cli.adoc    
+
+[id="using-command-completion_{context}"]
+= Using command completion
+
+[NOTE]
+====
+Currently command completion is only supported for bash, zsh, and fish shells.
+====
+
+{odo-title} provides a smart completion of command parameters based on user input. For this to work, {odo-title} needs to integrate with the executing shell.
+
+.Procedure
+
+* To install command completion automatically:
+.  Run:
++
+----
+$ odo --complete
+----
++
+.  Press `y` when prompted to install the completion hook.
+
+* To install the completion hook manually, add `complete -o nospace -C <full path to your odo binary> odo` to your shell configuration file. After any modification to your shell configuration file, restart your shell.
+
+* To disable completion, run:
++
+----
+$ odo --uncomplete
+----
+
```

Comment:
Add ```.  Press `y` when prompted to uninstall the completion hook.``` like it has been added in line no 24

Reply:
Added.